File: app/src/main/java/com/example/pam_project/features/tasks/list/TaskViewHolderPending.java
package com.example.pam_project.features.tasks.list;

import android.view.View;
import android.widget.ImageView;
import android.widget.TextView;

import androidx.annotation.NonNull;

import com.example.pam_project.R;

public class TaskViewHolderPending  extends TaskViewHolder {

    public TaskViewHolderPending(@NonNull View itemView) {
        super(itemView);
    }

    public void bind(final TaskInformation task) {
        super.bind(task);
        final TextView description = itemView.findViewById(R.id.description);
        description.setText(task.getDescription());
        final ImageView urgency = itemView.findViewById(R.id.urgency);
        if (task.getUrgency()) {
            urgency.setVisibility(View.VISIBLE);
        } else {
            urgency.setVisibility(View.GONE);
        }
    }
}

File: app/src/main/java/com/example/pam_project/features/tasks/list/TaskViewHolderPending.kt
package com.example.pam_project.features.tasks.list

import android.view.View
import android.widget.ImageView
import android.widget.TextView
import com.example.pam_project.R

class TaskViewHolderPending(itemView: View) : TaskViewHolder(itemView) {
    override fun bind(task: TaskInformation?) {
        super.bind(task)
        val description = itemView.findViewById<TextView>(R.id.description)
        description.text = task?.description
        val urgency = itemView.findViewById<ImageView>(R.id.urgency)
        if (task?.urgency == true) {
            urgency.visibility = View.VISIBLE
        } else {
            urgency.visibility = View.GONE
        }
    }
}